ConanInstall.cmake

```diff
@@ -57,9 +57,9 @@ if(NOT CONAN_OPENSTUDIO_ALREADY_RUN)
   conan_cmake_run(REQUIRES
     ${CONAN_READLINE}
     ${CONAN_QT}
-    openssl/1.1.0l # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
+    openssl/1.1.0l@nrel/testing # ruby 2.5.5 won't work with 1.1.1x, so use 1.1.0l here to try to force every package to align on the same as ruby
```

Comment:
@jmarrec can we switch these back to stable now?

Reply:
I think `openssl/1.1.0l` is stable and we can use this conan package.

Reply:
Yes